### DISCUSSION TITLE
Wrong path for static file and server actions

### DISCUSSION THREAD

**Discussion by laurent512** [vercel/next.js#63111] [App Router]
Hi, 

tldr; How can I read a local file (stored during build) in server actions using fs in production at vercel ? 

Description : 
I am working on a project to take a "template.docx" , update it, and store the result in my own google drive. 
Everything is working in development environment (on my machine) but there is a problem in production environment 

I am trying to read the file using  _fs.readFileSync(templatePath, "binary")_ 
All of the possible path I had in mind lead to  ENOENT: no such file or directory ( in vercel logs ) 
It is the same for me if the file is private of public, I tried both and I could not make it work. 

Would you be able to help me ? 

Thank you so much in advance,

Laurent
